docs/docker.md

```diff
@@ -67,40 +71,80 @@ There are three kinds of guidelines included:
     injected as part of the Docker build. Dockerfiles **SHOULD NOT** copy
     default configuration directly from GitHub.
 
--   A GE's Docker **MUST** be published at least under the 'fiware' account on
-    DockerHub (if possible, please avoid the term 'fiware' in your repository
-    name, as the username already contains it).
-
-The GE owner will be responsible for publication and maintenance operations. The
-publication method will be through the 'Automatic Build Procedure' which allows
-to link a GitHub repository to a Dockerhub one (and keep them in sync). In order
-to get access to that account and proceed with the publication, please get in
-touch with the FIWARE team at
-[fiware-help@lists.fiware.org](mailto:fiware-help@lists.fiware.org).
-
--   Dockerfiles and Dockerhub repositories **MUST** be linked from the FIWARE
-    Catalogue according to the Guidelines defined by the
-    [FIWARE Catalogue](http://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Working_with_the_FIWARE_catalogue#Creating_instances).
-
 -   The GitHub repository `README.md` **MUST** have a Docker reference - this is
     a link on the mandatory Docker Pulls `README.md` badge
 
 -   The **ReadtheDocs** GE Installation Documentation **MUST** include
     references to the Docker Hub image, and how to configure it.
 
+## Publication Requirements
+
+-   The GE owner will be responsible for Docker publication and maintenance
+    operations. Docker builds **SHOULD** be automated. GE Owner's build should
+    be triggered by git push, FIWARE's one will be triggered by the release of
+    new FIWARE-tagged versions.
+
 -   Docker containers **MUST** be tested before being published to the
     community. Error in Docker materials of a GE will be considered as critical
     and **MUST** be fixed immediately.
 
+-   Maintenance and release of interim point releases **MUST** be done using the
+    GE Owner own contributor account.
+
+-   The name of the `r/fiware/image` **MUST** match the standardized name of the
```

Comment:
This bullet is not a requirement for GE owners, so I'm not sure if it needs to be here.

Reply:
sounds good, yes